dotcom-rendering/src/layouts/TagPageLayout.tsx

```diff
@@ -77,6 +90,8 @@ export const TagPageLayout = ({ tagPage, NAV }: Props) => {
 	const inUpdatedHeaderABTest =
 		abTests.updatedHeaderDesignVariant === 'variant';
 
+	const contributionsServiceUrl = 'https://contributions.guardianapis.com'; // TODO: Read this from config (use getContributionsServiceUrl)
```

Comment:
Let's use the helper, which is used elsewhere -
https://github.com/guardian/dotcom-rendering/blob/3449102cba62a8b23ff5392e46cb77772a4132c0/dotcom-rendering/src/lib/contributions.ts#L237

Reply:
👍  Sounds good. The reason I left it hard-coded is that the helper does not expect the config to be of type `DCRTagPageType`, and it looks like it'll need a little work to make that fit. Would you be up for pairing on this?

Reply:
We decided to leave this for a future PR - the value isn't available in the data from frontend yet, so that needs to happen first.